### PR TITLE
schedule.rb changes

### PR DESCRIPTION
**Pull request by PJart**
update to schedule.rb, found some issues from testing

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -47,6 +47,7 @@ COPY --chown=gi-bill-data-service:gi-bill-data-service . .
 USER gi-bill-data-service
 RUN gem install bundler --no-document -v ${BUNDLER_VERSION}
 RUN bundle install --binstubs="${BUNDLE_APP_CONFIG}/bin" $bundler_opts && find ${BUNDLE_APP_CONFIG}/cache -type f -name \*.gem -delete
+RUN bundle exec whenever --update-crontab --set environment=test
```
Comment (ericboehs) [inline Dockerfile:50]: I believe this should be the production environment, yes?

[the remark this answers is not in the document]
Comment (PJart) [reply Dockerfile]: I updated it taking out the env

Comment (PJart) [reply Dockerfile]: I updated it taking out the env

Comment (zachclarity) [reply Dockerfile]: @rileyanderson  can you take a look at this.

Comment (zachclarity) [reply Dockerfile]: @PJart  see comments above.

Comment (PJart) [reply Dockerfile]: I updated the dockerfile to set it to production, by just setting it to bundle exec whenever --update-crontab, it sets the crontab to run without a specified env. The schedule commands run the commands to run the job.

Comment (PJart) [reply Dockerfile]: just to test it I can set the crontab in staging as well, we just need to test that the cron job can be run with the changes I made to schedule.rb

**Review by rileyanderson — APPROVED** [review]

**Pull request merged**